### PR TITLE
improved the content-changed the display location

### PR DESCRIPTION
**Pull request by KanerLev**

### CODE REVIEW

**Comment by Pratichhya** [comment]
@KanerLev, was this the display that was proposed https://eu-cdse.github.io/documentation/pr-preview/pr-288/Data/Sentinel2.html ? To be honest, it doesn't make sense to me why we are displaying Sentinel 2 L2A details before even description.

**Comment by KanerLev** [comment]
> @KanerLev, was this the display that was proposed https://eu-cdse.github.io/documentation/pr-preview/pr-288/Data/Sentinel2.html ? To be honest, it doesn't make sense to me why we are displaying Sentinel 2 L2A details before even description.

Yes. 
Although I agree with you, it is what is asked in CDSE-252.  CDSE-252 states: "It should be on the main page with **Sentinel-2** data description..."

**Comment by Pratichhya** [comment]
@KanerLev I updated the section as we discussed based on a suggestion from the internal ticket, that is I have included information for Sentinel2 L2A below the further details section.

Also, it was difficult to fix the markdown table, so I kept it as the HTML table. You can change it if you prefer the old one.  Also, let me know if you have additional comments on this or correct me if I missed any point or have any typo issues.

File: Data/Sentinel2_baseline.qmd [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  html:
+    code-fold: true
+execute:
+  echo: false
+  output: asis
+jupyter: python3
+
+---
+
+##### Additionally.....
```
Comment (KanerLev) [inline Data/Sentinel2_baseline.qmd:12]: I suggest to change the wording from "Additionally....." to "Sentinel-2 L2A Baselines" (as Sentinel-2 L2A Baselines was the header of the document to be published.

**Pull request merged**